pom.xml

```diff
@@ -44,7 +44,7 @@
     <publish_coordinator.yaml.file>${project.basedir}/src/main/resources/swagger.api/publications.yaml</publish_coordinator.yaml.file>
     <sharing_instances.yaml.file>${project.basedir}/src/main/resources/swagger.api/sharing_instance.yaml</sharing_instances.yaml.file>
     <!-- runtime dependencies -->
-    <folio-spring-base.version>7.1.0</folio-spring-base.version>
+    <folio-spring-base.version>7.1.1</folio-spring-base.version>
```

Comment:
To pick up the latest changes https://github.com/folio-org/folio-spring-support/pull/105 need to specify 7.2.0-SNAPSHOT version

Reply:
Fixed